modules/social_features/social_event/social_event.module

```diff
@@ -410,16 +412,24 @@ function social_event_preprocess_node(array &$variables) {
       '#view_mode' => $view_mode,
     ];
 
-    if ($view_mode === 'hero') {
-      // Add Enroll button.
-      if (empty($variables['event_enrollment'])) {
-        $form = \Drupal::formBuilder()
-          ->getForm('Drupal\social_event\Form\EnrollActionForm');
-        $render_array = [
-          'enroll_action_form' => $form,
-        ];
-        $variables['event_enrollment'] = $render_array;
-      }
+    switch ($view_mode) {
+      case 'hero':
+        // Add Enroll button.
+        if (empty($variables['event_enrollment'])) {
+          $form = \Drupal::formBuilder()
+            ->getForm('Drupal\social_event\Form\EnrollActionForm');
+          $render_array = [
+            'enroll_action_form' => $form,
+          ];
+          $variables['event_enrollment'] = $render_array;
+        }
+
+        break;
+
+      case 'featured':
+        $variables['#cache']['max-age'] = 0;
```

Comment:
Hi @VBaldych,
It's not a good practice to add max-age 0. 
We need to check if we have other ways to solve this issue.
For example check `social_event_node_view_alter()` where we add some cache tags and context to the even teaser.

Also we need to compare featured view mode with normal teaser.
Srtrange if we have issue only in featured.